packages/app/src/cli/models/app/loader.ts

```diff
@@ -60,9 +60,7 @@ async function loadConfigurationFile(
     }
   }
   // Convert snake_case keys to camelCase before returning
-  return {
-    ...Object.fromEntries(Object.entries(configuration).map((kv) => [camelize(kv[0]), kv[1]])),
-  }
+  return convertKeysToCamelCase(configuration)
```

Comment:
sorry, I don't think we can do yet. There are extensions we were working around and we'll need to check that those still work